cli/dcoscli/job/main.py

```diff
@@ -228,6 +233,39 @@ def _remove(job_id, stop_current_job_runs=False):
     return 0
 
 
+def _queue(job_id, json_flag=False, quiet=False):
```

Comment:
I don't understand what quiet means and it is not documented even in the other usages, so can you maybe document it?

Reply:
quiet or q is docker inspired... docker ps -lq returns just the sha of the last run container.  I've been adding these in a number of places in the CLI to make them more composable.. it is step 1... in other words you should be able to do this `dcos job kill jobID $(dcos job queue -q)` and it will kill all the jobs in the queue for the jobID.

Reply:
oh ok, so can we add a documentation for this somewhere? At least one sentence as it is in https://docs.docker.com/engine/reference/commandline/ps/